google/resource_container_cluster.go

```diff
@@ -310,6 +310,7 @@ func resourceContainerCluster() *schema.Resource {
 						},
 					},
 				},
+				DiffSuppressFunc: emptyOrDefaultStringSuppress("PROVIDER_UNSPECIFIED"),
```

Comment:
I believe the DiffSuppressFunc should be on the `provider` field under `network_policy`. Could you test quickly with a simple config that it works and reports to the PR about your test. Thanks :)

Reply:
I don't know how it worked before - anyways, done! I added some test steps to the network policy test as well.